ir_datasets/datasets/trec_fair_2021.py

```diff
@@ -0,0 +1,179 @@
+import json
+import mwparserfromhell
+import codecs
+from typing import NamedTuple, Dict
+import ir_datasets
+from ir_datasets.util import GzipExtract, Cache, Lazy
+from ir_datasets.datasets.base import Dataset, YamlDocumentation, FilteredQueries
+from ir_datasets.formats import BaseQueries, BaseDocs, BaseQrels, TrecQrel
+from ir_datasets.indices import PickleLz4FullStore
+import pyautocorpus
+from itertools import chain
+
+
+_logger = ir_datasets.log.easy()
+
+
+NAME = 'trec_fair_2021'
+
+
+class FairTrecDoc(NamedTuple):
```

Comment:
A few changes I'm thinking here:
 - `geographic_locations` -> `Optional[List[str]]`, matching the type in metadata
 - `quality_score` -> `Optional[float]`
 - `quality_score_disc` -> `Optional[str]`
 - Change "NA"s to `None` when metadata is not available